launcher/LoggedProcess.cpp

```diff
@@ -66,13 +66,12 @@ QStringList LoggedProcess::reprocess(const QByteArray& data, QTextDecoder& decod
     }
 
 #if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
```

Comment:
The if isn't needed

Reply:
Oh yeah, the two branches are the same now...